src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -572,7 +572,7 @@ private async Task CheckJobCompletionStatus()
                             OperationOutcomeConstants.IssueSeverity.Error,
```

Comment:
Nit: Since message isn't used separately now, could it be combined with userMessage into one variable?

Reply:
Message combined with userMessage